at-date-console/src/main/kotlin/com/amerharb/atdate/Main.kt

```diff
@@ -10,14 +10,16 @@ fun main(args: Array<String>) {
 	if (args.isEmpty()) {
 		println("enter @...@ to encode, 0x... to decode, C to copy last result or Q to quit")
 		while (true) {
```

Comment:
can be simplified to 
while (mainMenu()) {}
println("Exiting 👋...")

Reply:
done